fog/ledger/server/tests/store.rs

```diff
@@ -242,7 +241,7 @@ pub fn direct_key_image_store_check(logger: Logger) {
     // AuthResponseOutput
     let auth_message = attest::AuthMessage::from(client_auth_response);
     // Initiator accepts responder's message.
-    let auth_response_event = AuthResponseInput::new(auth_message.into(), Verifier::default());
```

Comment:
I understand why this works, but I find it somewhat counter-intuitive. A default verifier letting everything verify makes sense to me, however my expectation would be that if an empty array of measurements results in the measurement obtained from the enclave not matching anything in the (empty) array and the verification will fail. I guess it's just weird to me that some methods, when called, force a verification of some sort (all of them, I think?) but the one that receives an array essentially doesn't do anything if the array is empty.
I am not sure it's worth addressing or what a good solution would be (other than `Option<Vec<...>>` which is going to be annoying, maybe made a bit better by `into Option<...>`).

Reply:
I guess another way to phrase this is that maybe it makes it a bit too easy for users to implicitly skip verification? `&[]` is perhaps a bit less explicit than a default Verifier that is not configured to verify anything?

I'm fine leaving it as is, but would like to hear @cbeck88 's opinion on this as well.

Reply:
I agree an empty array meaning "don't check measurements" seems like a bad API.

I created #3374, It's title was "Handle report only interfaces", I updated name to emphasize the potential footgun here rather than the current non test usage of no measurements.